app/Http/Controllers/Frontend/Export/ExportController.php

```diff
@@ -4,8 +4,10 @@
 
 use App\Http\Controllers\Backend\Export\ExportController as ExportBackend;
 use App\Http\Controllers\Controller;
+use App\Models\User;
 use Carbon\Carbon;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
```

Comment:
Das braucht es hier nicht, oder? Also das hat nichts damit zu tun?
```suggestion
use Carbon\Carbon;
use Illuminate\Http\Request;
```